code/datums/traits.dm

```diff
@@ -1072,6 +1072,7 @@ obj/trait/pilot
 	points = 0
 	isPositive = 1
 	unselectable = 1
+	var/onlifeprob = 35
```

Comment:
don't like splitting this out to a var only used once and not modified, make it const or remove

Reply:
done